e2e_mysql_test.go

```diff
@@ -32,7 +32,7 @@ var (
 	// "Cloud SQL MySQL instance connection name, in the form of 'project:region:instance'.
 	mysqlConnName = os.Getenv("MYSQL_CONNECTION_NAME")
 	// "Cloud SQL MySQL instance connection name, in the form of 'project:region:instance'.
-	mysqlIAMConnName = os.Getenv("MYSQL_IAM_CONNECTION_NAME")
+	mysqlIAMConnName = os.Getenv("MYSQL_CONNECTION_NAME")
```

Comment:
nit: should we not just get rid of `mysqlIAMConnName` as it will be identical to `mysqlConnName`?